drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/projects_importer.rb

```diff
@@ -98,10 +101,18 @@ def start
     def sanity_check
       msg = []
 
-      msg << 'Funder.csv was not present.' unless File.exist?("#{dir}/Funder.csv")
-      msg << 'Organization.csv was not present.' unless File.exist?("#{dir}/Organization.csv")
-      msg << 'Project.csv was not present.' unless File.exist?("#{dir}/Project.csv")
-      msg << 'Inventory.csv was not present.' unless File.exist?("#{dir}/Inventory.csv")
+      [
+        'Funder.csv',
+        'Organization.csv',
+        'Project.csv',
+        'Inventory.csv',
+        'ProjectCoC.csv',
+        'HmisParticipation.csv',
+        'CeParticipation.csv',
+        'ProjectUnitTypes.csv',
+      ].each do |file|
```

Comment:
Also feels like we could expand this to a hash of 
```
{
  'Funder.csv' => GrdaWarehouse::Hud::Funder,
  ...
}
```
and use it above where we call `upsert_hud_file(...)` , though I do see there's one difference, so we'd need a special case...

Reply:
eh I am going to by happy with the DRY-ing I did so far